pkgs/by-name/ve/vegur/package.nix

```diff
@@ -5,15 +5,21 @@
 }:
 
 let
+  inherit (lib.strings)
+    replaceStrings
+    ;
+
   majorVersion = "0";
   minorVersion = "701";
 in
-stdenvNoCC.mkDerivation {
+stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "vegur";
   version = "${majorVersion}.${minorVersion}";
 
   src = fetchzip {
-    url = "https://dotcolon.net/download/fonts/vegur_${majorVersion}${minorVersion}.zip";
+    url = "https://dotcolon.net/files/fonts/vegur_${
+      replaceStrings [ "." ] [ "" ] finalAttrs.version
```

Comment:
Or make major/minorVersion part of the attrs? We can save the replaceStrings trick.

Reply:
Good idea. Updated with suggestion.